test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -86,11 +80,28 @@ public void It_migrates_dotnet_new_console_with_identical_outputs()
             VerifyAllMSBuildOutputsRunnable(projectDirectory);
         }
 
-        [Fact(Skip="https://github.com/dotnet/cli/issues/4299")]
+        [Fact]
         public void It_migrates_dotnet_new_web_with_outputs_containing_project_json_outputs()
         {
-            var projectDirectory = Temp.CreateDirectory().Path;
-            var outputComparisonData = GetDotnetNewComparisonData(projectDirectory, "web");
+            var globalDirectory = Path.Combine(AppContext.BaseDirectory, "newwebtest");
+            var projectDirectory = Path.Combine(globalDirectory, "project");
+            if (Directory.Exists(projectDirectory))
+            {
+                Directory.Delete(projectDirectory, true);
+            }
+            Directory.CreateDirectory(projectDirectory);
+
+            WriteGlobalJson(globalDirectory);
+            DotnetNew(projectDirectory, "web");
```

Comment:
This won't work for long. Should we have a web test asset to be used here? This is just adding debt for when we switch the templates.